packages/middleware-log-errors/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@dotcom-reliability-kit/middleware-log-errors",
+  "version": "0.0.0",
+  "description": "Express middleware to consistently log errors",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Financial-Times/dotcom-reliability-kit.git",
+    "directory": "packages/middleware-log-errors"
+  },
+  "homepage": "https://github.com/Financial-Times/dotcom-reliability-kit/tree/main/packages/middleware-log-errors#readme",
+  "bugs": "https://github.com/Financial-Times/dotcom-reliability-kit/issues",
+  "license": "MIT",
+  "engines": {
+    "node": "14.x || 16.x",
+    "npm": "7.x || 8.x"
+  },
+  "main": "lib",
+  "dependencies": {
+    "@dotcom-reliability-kit/serialize-error": "^0.1.1",
+    "@dotcom-reliability-kit/serialize-request": "^0.1.0",
+    "@financial-times/n-logger": "^10.1.0"
```

Comment:
This could feasibly be a wider range as we're only using the error method and it's signature has barely changed.

This would help to avoid the application having a different version of logger for application code.

Reply:
Great spot 👍  I'll have a look at what versions our apps use and widen the range to make sure nobody will _have_ to update n-logger to use this package

Reply:
Fixed in a new commit which switches to a range of `>=6.0.0 <11.0.0`

> I had a look across all n-logger references in `package.json` files
across the FT. It seems like nothing in Customer Products is using a
version lower than `6.x`. The only breaking changes between this
version and `10.x` are dropping support for Node.js versions - the API
doesn't change at all.